VideoWeb/VideoWeb/Mappings/BookingForHostResponseMapper.cs

```diff
@@ -20,23 +20,20 @@ public BookingForHostResponseMapper(IMapTo<ParticipantForHostResponse, VideoWeb.
     public Contract.Responses.ConferenceForHostResponse Map(ConfirmedHearingsTodayResponse booking, List<ConferenceForHostResponse> conferences)
     {
         var conference = conferences.Find(x => x.HearingId == booking.Id);
-        var dto = new Contract.Responses.ConferenceForHostResponse
-        {
-            Id = conference.Id,
-
-            CaseName = booking.CaseName,
-            CaseNumber = booking.CaseNumber,
-            CaseType = booking.CaseTypeName,
-            ScheduledDuration = booking.ScheduledDuration,
-            ScheduledDateTime = booking.ScheduledDateTime,
-            NumberOfEndpoints = booking.Endpoints.Count,
-            HearingVenueIsScottish = booking.IsHearingVenueScottish,
-            Status = Enum.Parse<ConferenceStatus>(conference.Status.ToString()),
-            ClosedDateTime = conference.ClosedDateTime,
-            Participants =
-                conference.Participants.Select(_participantForHostResponseMapper.Map)
-                    .ToList() // need to use participant list from video api to include QL users
-        };
+        var dto = new Contract.Responses.ConferenceForHostResponse();
```

Comment:
are these changes needed? Probably best to use the object initialiser if we can

Reply:
I actually prefer it this way, because if it throws an exception you can see immediately which line it was on, and which property has the null ref, rather than it just pointing to line 23, and they you have to work it out. Would save debug time if we didnt use object initialisers everywhere

Reply:
understandable